modules/images/dominant-color/class-dominant-color-image-editor-gd.php

```diff
@@ -1,7 +1,7 @@
 <?php
 /**
  * WordPress Image Editor Class for Image Manipulation through GD
- * with dominant color detection
+ * with dominant color detection.
```

Comment:
There should be periods in most docs, functions, methods, parameters, etc. But the file header is one exception to the rule, so I think this should be left as it was.

Reply:
Thank you for the information on documentation standards.